sklearn/ensemble/_stacking.py

```diff
@@ -0,0 +1,654 @@
+            self.transform(X), **predict_params
+        )
+
+
+class StackingClassifier(_BaseStacking, ClassifierMixin):
+    """Stacked of estimators using a final classifier.
+
+    Stacked generalization consists in stacking the output of individual
+    estimator and use a classifier to compute the final prediction. Stacking
+    allows to use the strength of each individual estimator by using their
+    output as input of a final estimator.
+
+    Note that `estimators` are fitted on the full `X` while `final_estimator`
+    is trained using cross-validated predictions of the base estimators using
+    `cross_val_predict`).
+
+    .. versionadded:: 0.22
+
+    Read more in the :ref:`User Guide <stacking>`.
+
+    Parameters
+    ----------
+    estimators : list of (string, estimator) tuples
+        Base estimators which will be stacked together. An estimator can be set
+        to None or 'drop' using `set_params`.
+
+    final_estimator : estimator object, default=None
+        A classifier which will be used to combine the base estimators.
+        The default classifier is a `LogisticRegression`.
+
+    cv : int, cross-validation generator or an iterable, default=None
+        Determines the cross-validation splitting strategy. Possible inputs for
+        cv are:
+
+        * None, to use the default 5-fold cross validation,
+        * integer, to specify the number of folds in a (Stratified) KFold,
+        * An object to be used as a cross-validation generator,
+        * An iterable yielding train, test splits.
+
+        For integer/None inputs, if the estimator is a classifier and y is
+        either binary or multiclass, `StratifiedKFold` is used. In all other
+        cases, `KFold` is used.
+
+        Refer :ref:`User Guide <cross_validation>` for the various
+        cross-validation strategies that can be used here.
+
+    predict_method : list of string or 'auto', default='auto'
+        Methods called for each base estimator. It can be:
+
+        * if a list of string in which each string is associated to the
+          `estimators`,
+        * if 'auto', it will try to invoke, for each estimator,
+          `predict_proba`, `decision_function` or `predict` in that order.
+
+    n_jobs : int, default=None
+        The number of jobs to run in parallel for both `fit` the `estimators`.
+        `None` means 1 unless in a `joblib.parallel_backend` context. -1 means
+        using all processors. See Glossary for more details.
+
+    random_state : int, RandomState instance or None, optional (default=None)
+        If int, random_state is the seed used by the random number generator;
+        If RandomState instance, random_state is the random number generator;
+        If None, the random number generator is the RandomState instance used
+        by `np.random`. Used to set the `cv`.
+
+    Attributes
+    ----------
+    estimators_ : list of estimator object
+        The base estimators fitted.
```

Comment:
I find the use of "base estimator" a touch confusing here. Why not just "The elements of the estimators parameter, having been fitted on the training data."

Also, no mention here of the 'drop' handling.